frontend/src/components/inputs/DateTime.vue

```diff
@@ -14,10 +16,28 @@ export default {
   name: "DateTime",
   components: {BaseInput},
   props: ["label", "modelValue", "args", "vertical", "required"],
+  computed: {
+    value() {
+      return this.modelValue && this.convertToDateTimeLocalString(this.modelValue);
+    }
+  },
   methods: {
     onInput(event) {
       this.$emit("update:modelValue", event.target.value);
     },
+    convertToDateTimeLocalString(date) {
+      if (!(date instanceof Date)) {
+        date = new Date(date)
+      }
+
+      const year = date.getFullYear();
+      const month = (date.getMonth() + 1).toString().padStart(2, "0");
+      const day = date.getDate().toString().padStart(2, "0");
+      const hours = date.getHours().toString().padStart(2, "0");
+      const minutes = date.getMinutes().toString().padStart(2, "0");
+
+      return `${year}-${month}-${day}T${hours}:${minutes}:00`;
```

Comment:
If the [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local#value) are to be trusted, the expected format for the return value is `YYYY-MM-DDThh:mm`. So we could simplify to:

```suggestion
      return date.toISOString().substring(0, 16);
```

Assuming that you don't find this too opaque.

Reply:
I like it, it's even more readable. Thanks!